### ISSUE TITLE
Collect survey results and do some research using dimension reduction

### ISSUE THREAD

**Issue by Kalethars** [LeftValues/leftvalues.github.io#55]
I think if we need to make this test more accurate, we need to analyze it in a more quantative way. We can mark from 2 to -2 for each selection, and use pca to extract the 8 most influential factors and see how questions contribute to these factors. And thus we can find out the meaning of these factors.

**Comment by asycns** [comment]
Yes, I think so. Probably there are some techniques that we can collect data from participants. Reliability and validability of this questionnaire should be calculated.
